python/metaparticle_pkg/builder/docker_builder.py

```diff
@@ -3,7 +3,7 @@
 
 from docker import APIClient
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger('.'.join(__name__.split('.')[:-1]))
```

Comment:
OK, that gets us common logging for builder and runner. But this is a specific choice, so it would be nice to have a comment line indicating the purpose.

Reply:
Makes sense, I'll make the adjustment